### DISCUSSION TITLE
Connecting to NATS cluster

### DISCUSSION THREAD

**Discussion by alberk8** [nats-io/nats.net#446] [Q&A]
How does the client connect to a 3 or 5 node NATS cluster?  Do I need something like a load balancer like nginx or traefik in front of the cluster?   The nats option only support a single url endpoint.

```
 var opt = new NatsOpts
 {
     Url = "nats://192.168.2.5:4222",
     Name = "Nats_Connect",
       
 };
```

**Comment by mtmk** [comment]
you should be able to add a comma separated list of URLs e.g.:
```
Url = "nats://nats1.example.net,nats://nats2.example.net,nats://nats3.example.net",
```
